tests/lib/snaps/store/test-snapd-curl/snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: test-snapd-curl
+version: '7.58'
+summary: Curl is a command-line tool for transferring data.
+description: |
+  Curl is a command-line tool for transferring data specified with URL syntax.
+grade: stable
+confinement: strict
+base: core18
```

Comment:
should this use core20 base maybe?

Reply:
It is failing to run the snap. I'll update this in a following pr.

$ test-snapd-curl.curl --version
/snap/test-snapd-curl/x1/usr/bin/curl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /snap/test-snapd-curl/x1/usr/lib/x86_64-linux-gnu/libldap_r-2.4.so.2)
/snap/test-snapd-curl/x1/usr/bin/curl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /snap/test-snapd-curl/x1/usr/lib/x86_64-linux-gnu/liblber-2.4.so.2)
/snap/test-snapd-curl/x1/usr/bin/curl: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /snap/test-snapd-curl/x1/usr/lib/x86_64-linux-gnu/libsqlite3.so.0)
/snap/test-snapd-curl/x1/usr/bin/curl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /snap/test-snapd-curl/x1/usr/lib/x86_64-linux-gnu/libsqlite3.so.0)